### PR TITLE
First draft of Redis blog post.

### PR DESCRIPTION
**Pull request by jkodroff**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Comment by scottslowe** [comment]
@jkodroff Is there a reason you're working from a personal fork of `pulumi-hugo`, instead of from this repo directly? Using a fork won't generate a preview URL, which can make it easier to review blog posts sometimes.

File: themes/default/content/blog/redis-cloud-provider/index.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+{{% notes type="info" %}}
+The Redis Cloud provider does not support Fixed subscriptions. You'll need to create a Flexible subscription in order to use the provider to manage resources with Pulumi.
+{{% /notes %}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as rediscloud from "@rediscloud/pulumi-rediscloud";
+
+const card = rediscloud.getPaymentMethodOutput({
```
Comment (scottslowe) [inline themes/default/content/blog/redis-cloud-provider/index.md:67]: Would it be better/is it possible to store these values as configuration values/secrets? I know it's only the last four digits of a credit card, but I still hate seeing that information in a program.

File: themes/default/content/blog/redis-cloud-provider/index.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+* An AWS VPC across 3 Availability Zones, with public and private subnets.
+* VPC Peering to enable connectivity between the Redis cluster and the VPC.
+
+You'll also prove the architecture works by spinning up an EC2 instance and running simple commands that target the Redis cluster.
+
+The following diagram shows the major components of the architecture:
+
+![Architecture diagram of an AWS VPC across 3 availability zones, connected to Redis Enterprise Cloud via VPC peering](architecture.png)
+
+## First Steps, Redis Enterprise Cloud Resources and Peering
```
Comment (scottslowe) [inline themes/default/content/blog/redis-cloud-provider/index.md:39]:
```suggestion
## Redis Enterprise Cloud Resources
```

**Review by scottslowe — COMMENTED** [review]
A few suggestions to consider

File: themes/default/content/blog/redis-cloud-provider/index.md (hunk 1 above)
Comment (jkodroff) [reply themes/default/content/blog/redis-cloud-provider/index.md:67]: This is intentional. The corresponding example in Pulumi/examples has it parameterized, but for easier reading I decided to hardcode it here. I explain in a note that there are subtle differences between the example and post code as well.

**Comment by jkodroff** [comment]
No, definitely not intentional to use a fork. Will address any in-flight feedback, close this PR, and recreate. BLARGH.

File: themes/default/content/blog/redis-cloud-provider/index.md (hunk 2 above)
Comment (jkodroff) [reply themes/default/content/blog/redis-cloud-provider/index.md:39]: Fixed.

**Comment by jkodroff** [comment]
Closing. Creating a new PR not off a fork.

**Pull request closed without merging**